en/SUMMARY.md

```diff
@@ -538,8 +537,7 @@
       * [wind](msg_docs/wind.md)
       * [yaw_estimator_status](msg_docs/yaw_estimator_status.md)
     * [MAVLink Messaging](middleware/mavlink.md)
-    * [RTPS/DDS Interface: PX4-Fast RTPS(DDS) Bridge](middleware/micrortps.md)
```

Comment:
Deleted link. Doc still exits, but just redirects to the DDS doc. That has a note explaining where FastRTPS went.